### DISCUSSION TITLE
Slowonly is better than SlowFast

### DISCUSSION THREAD

**Discussion by JasseurHadded1** [open-mmlab/mmaction2#2471] [General]
Hello,

I have seen in the original paper of slowfast that the accuracy difference between slowonly and slowfast is just 3%, I have tested that with my own dataset and I have found close results, then I have trained slowonly but with 32 frames in input instead of 4, and it gave a better result than slowfast.

I have just wanted to discuss that with you, do you see this as normal result? and is there any theoretical explanation?

Thanks,

**Comment by JasseurHadded1** [comment]
My dataset is a hand gesture dataset (Ipn hand)

**Comment by hukkai** [comment]
You are right. ShowOnly models can get the same performance as their SlowFast counterparts when carefully tuned. For example https://github.com/open-mmlab/mmaction2/tree/main/configs/recognition/slowonly
